lib/common/src/Dom/Document.php

```diff
@@ -1358,17 +1370,14 @@ private function restoreAmpEmojiAttribute($html)
      *
      * ⚠️ This is not cryptographically secure!
      *
-     * @param int $min Lower limit for the generated number
-     * @param int $max Upper limit for the generated number
-     * @return int A random number between min and max
+     * @return int A random number.
      */
-    private function rand($min = 0, $max = 0)
```

Comment:
That was probably meant to be `$min = null, $max = null` ... ? 🤦

Reply:
I've opted now to remove the `rand` method altogether.